packages/tx-construction/test/tx-builder/TxBuilder.test.ts

```diff
@@ -414,15 +412,7 @@ describe('GenericTxBuilder', () => {
 
     it('throws RewardAccountMissingError error if no reward accounts were found', async () => {
       txBuilderProviders.rewardAccounts.mockResolvedValueOnce([]);
-
-      try {
-        await txBuilder.delegate(poolId).build().inspect();
-      } catch (error) {
-        const buildErrors = error as TxBodyValidationError[];
-        expect(buildErrors?.length).toBe(1);
-        expect(buildErrors[0] instanceof RewardAccountMissingError).toBeTruthy();
-      }
-      expect.assertions(2);
+      expect(txBuilder.delegate(poolId).build().inspect()).rejects.toThrowError(RewardAccountMissingError);
```

Comment:
it's curious this works. I thought you need to [await it](https://jestjs.io/docs/tutorial-async#asyncawait):
```diff
-      expect(txBuilder.delegate(poolId).build().inspect()).rejects.toThrowError(RewardAccountMissingError);
+      await expect(txBuilder.delegate(poolId).build().inspect()).rejects.toThrowError(RewardAccountMissingError);

Reply:
Nice catch, not awaiting could result in false positives 68fe2682bd0895475c8e868b489ddf2ff8fef98b